test_data/locations_data.json

```diff
@@ -0,0 +1,17 @@
+{
+    "table_name": "locations",
+    "data": [
+        {
+            "name":"Kampala",
+            "abbreviation":"KLA"
+        },
+        {
+            "name": "Nairobi",
+            "abbreviation": "NBO"
+        },
+        {
+            "name": "Lagos",
+            "abbreviation": "LOS"
+        }
```

Comment:
This should have Kigali also as one of the locations.
`        {
             "name": "Kigali",
             "abbreviation": "KGL"
         },`

Reply:
okay @shaluchandwani let me add it.